.github/workflows/main.yml

```
@@ -10,7 +10,36 @@ on:
 
 jobs:
   test:
-    name: "Python ${{ matrix.python }}, Test: ${{ matrix.lang }}"
+    name: "Python ${{ matrix.python }} API tests"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python:
+          - 3.6
+          - 3.7
+          - 3.8
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 5
+      - name: Setup Python ${{ matrix.python }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Setup dependencies and run API tests
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          TEST: API
+          PYTHON: ${{ matrix.python }}
+        run: |
+          export BUILD_DIRECTORY=$GITHUB_WORKSPACE
+          bash $GITHUB_WORKSPACE/.ci/setup.sh
+          bash $GITHUB_WORKSPACE/.ci/test.sh
+
+  e2e-test:
+    name: "Python ${{ matrix.python }}, Test: '${{ matrix.lang }}'"
```

We can use this value without escaping. Or were these quotes added just for aesthetic purposes?
```suggestion
    name: "Python ${{ matrix.python }}, Test: ${{ matrix.lang }}"
```

---

Yeah - the latter :D Reverting it back